Cargo.toml

```diff
@@ -89,4 +89,5 @@ debug = true
 debug = true
 
 [patch.crates-io]
-halo2 = { git = "https://github.com/zcash/halo2.git", rev = "f9b3ff2aef09a5a3cb5489d0e7e747e9523d2e6e" }
+# halo2_proofs = { git = "https://github.com/zcash/halo2.git", branch = "halo2_gadgets-import" }
+# halo2_gadgets = { git = "https://github.com/zcash/halo2.git", branch = "halo2_gadgets-import" }
```

Comment:
This cherry-picked commit is now technically broken, because it doesn't have any patches active. I don't mind _too_ much if it stays broken, but if you're going to rebase this PR, then please fix it to match what is in #273.